static-assets/components/cstudio-contextual-nav/wcm-site-dropdown-mods/wcm-root-folder.js

```diff
@@ -1015,6 +1015,11 @@
           $sidebarHighlight.hide();
         });
 
+        $('#acn-dropdown-menu').on('scroll', function(){
```

Comment:
Currently if you scroll the doc, the highlight and ellipsis gets mispositioned. Will this work if the non-sidebar area gets scrolled?

Reply:
This event fixes the issue when scrolling in the tree area, for the other issue (scrolling on the doc where the highlight gets mispositioned there's another fix, where the retrieval of the position of the element being highlighted is relative to the window instead of the doc. Please check this video: https://www.loom.com/share/a938978368d543edacefed80427dea1b

Reply:
I can see you're directly scrolling the sidebar. We need to test scrolling the main document without leaving the sidebar.

When you continue to scroll past the scroll on the sidebar (at least o mac/chrome) it will start scrolling the main document. Please check if things behave as they should in that case.

Let me know if that works.

Reply:
Tested that scenario, when scrolling the main document (past the scroll on the sidebar) highlight doesn't break